viewers/wicket/ui/src/main/java/org/apache/isis/viewer/wicket/ui/pages/bootstrap-overrides-all-v2.css

```diff
@@ -979,6 +979,10 @@ li.select2-results__option {
     padding-right: 15px;
 }
 
+.multiLineStringPanel .scalarValueInlinePromptLink textarea {
+    background-color: white;
```

Comment:
It might not be a good idea to hard-code a color value that way, as there are different bootstrap themes one can choose to use. There are also fancy 'dark' themes that would require the 'opposite' color.